index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="box header">
+            <span>My Logo</span>
+            <span><img src="img/Enabled.png"></span>
+        </div>
+        <div class="box content">
+            <div class="title">
+            I’m Mariane 
+            Glad to see 
+            you!
+            </div>
```

Comment:
- [ ] I suggest using `<h1 class="title">` tag instead of `<div>` tag, because this is the main title and it's better for SEO 💯
> If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.